### ISSUE TITLE
Add new Onboarding Email for Camps

### ISSUE THREAD

**Issue by viandrea** [openLAMA/Welcome#98]
As an Uni Admin, I want to be able to send a follow-up email to the org type CAMPS so that they have the important testing details and attachments.

**Email and Attachements**
https://drive.google.com/drive/folders/19BZGj0KcmPYqh8rn7IyuvedtgmiuNYVL?usp=sharing

**Comment by viandrea** [comment]
@TsvyatkoIvanov @tockata 

The email template is not ready yet.

@tockata  Would it be possible to add a specific follow-up email and attachments for the "Camps"? Attachments are completely different from what we have now for SME and companies.

**Comment by tockata** [comment]
@viandrea 
Yes, we can.
We need:
- the attachments;
- the template with placeholders for the data we should insert
- the conditions or validations that we have to apply to before sending the email.

**Comment by tockata** [comment]
@viandrea,
${numberOfTestParticipants} is the "NumberOfSamples" column in DB /not "RegisteredEmployees" one/, right?

**Comment by viandrea** [comment]
Correct, number of samples that we are collecting during the registration. 

For camps we wont have the numbers for registered people,  similar as for schhols

**Comment by tockata** [comment]
@viandrea please send me the Subject of the email, thanks!

**Comment by tockata** [comment]
@viandrea,
${OrgContactPersonName}, ${OrgContactPersonEmail} - 
Will this be selected/filled by from the user who sends the onboarding email or if not - which contact we should take if the organization has more then 1 contacts(users)?

**Comment by viandrea** [comment]
I think there will be always one contact.

**Comment by tockata** [comment]
So it is ok to take the first contact and put it there.
It will not be a free text filled by the user who send the onboarding email.

**Comment by TsvyatkoIvanov** [comment]
I was thinking of having a dropdown where they can select who is the contact person of the organization. 
If we are gonna choose the first one, i still need to show to who this person is, like the name and email.
But if i show the first person from the list of contacts, i can't guarantee that the database will get the same person.
I need to send this person to the API. 
I already created the auto complete dropdown, but i can remove it and just show the first person i get in the contacts array.

Also, currently we still have the additional contact input,
and the receivers array, with validation that at least one of the fields must have a value.

I can remove these completely, or just remove the required validation ?

**Comment by viandrea** [comment]
Dropdown is a great idea!

No manual user input freetext field in this case
